### PR TITLE
Style guide updates

### LINKED ISSUE #511: Add best practice for negative examples to style guide
E.g., for `StreetAddress`:

```
gist:negativeExample "A PO box address or FPO code."
```

Evolved from the discussion of PR #508.

@uscholdm I don't see a similar issue from you, but if you have submitted one we can close this as a duplicate.

### LINKED ISSUE #966: Define best practice for using or not using ontology terms in annotations
For example, the definition of `gist:OrderedCollection` is

```
skos:definition "A collection in which the members are sequentially ordered. All members of an OrderedCollection are OrderedMembers."^^xsd:string ;
```

I prefer the use of ordinary English language instead, or at the minimum consistency. (This one is inconsistent because "collection" is not referred to by the class name "Collection."). In this case:

```
skos:definition "A collection in which the members are sequentially ordered. All members of an ordered collection are ordered members."^^xsd:string ;
```

### LINKED ISSUE #1080: Broken link at gistStyleGuide.md
Maybe you could replace [1] by [2] at "The notion of rigid classes originates in OntoClean" [3] because [1] throws "{"message":"Token is required"}".

[1] https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.99.7618&rep=rep1&type=pdf
[2] https://scholar.archive.org/work/na4kfdl4hfczjgbwwnportg46y
[3] https://github.com/semanticarts/gist/blob/develop/docs/gistStyleGuide.md

### LINKED ISSUE #1086: Add section on inverses to gist style guide
* Don't use them
* Choose direction based on cardinality (parent to child)

### PR DESCRIPTION
**Pull request by rjyounes**
Closes #511 - Best practices for negative examples in `skos:example`
Closes #966 - Using or not using ontology terms in annotations
Closes #1080 - Fixed broken link to OntoClean
Closes #1086 - Best practices for inverses (don't)

### CODE REVIEW

File: docs/gistStyleGuide.md [hunk 1]
```diff
@@ -108,13 +110,13 @@ These conventions apply to both data and taxonomy terms.
...
 
 For example, given the class hierarchy `Living Thing` > `Person` > `Professor`, where the first two classes are rigid and the third is not, the name for Sir Tim Berners-Lee is `_Person_Sir_Tim_Berners_Lee`.
 
-Exceptions to this guideline arise may arise in IRI minting during data transformation. Based on how the data is presented, it is often difficult or cumbersome to know the most specific type of an instance, so one can fall back on a higher-level class. E.g., `_Magnitude_` rather than `_Duration_`.
+Exceptions to this guideline arise may arise in IRI minting during data transformation. Based on how the data is presented, it is often difficult or cumbersome to know the most specific type of an instance, so one can fall back on a higher-level class. E.g., when processing a table of organizations, it may not be possible to know which are governmental organizations, so the infix `_Organization_` can be used throughout.
 
 Note: As of version 12.0.0, gist itself does not itself follow the infix convention, though it does use the leading underscore. This is under consideration for a future update.
```
Comment (dylan-sa) [inline docs/gistStyleGuide.md:121]: After the unit instances are removed in 13.0.0, will we have any remaining violations of this guideline (maybe the media types)? If not, we can remove this. If yes, we can bump to 13.0.0.

**Review by dylan-sa — CHANGES_REQUESTED** [review]

File: docs/gistStyleGuide.md [hunk 2]
```diff
@@ -231,10 +235,39 @@ Certain RDFS annotations are recommended where there is no SKOS equivalent.
...
+
+- Reduce cognitive load for developers and implementers of the ontology.
+- Promote uniformity in the graph.
+- Eliminate the need for duplicate query paths in queries.
+- Reduce memory load during inferencing.
+- Simplify validation and writing and maintaining SHACL shapes.
+
+In selecting which of a potential pair of inverses to define, we apply the child-to-parent or cardinality principle: select the direction which will generally produce the fewest query results. Examples:
+
+| Child | Parent |
```
Comment (uscholdm) [inline docs/gistStyleGuide.md:262]: Suggest:

`| Child to Parent (used) | Parent to Child (not used)|`

COme to think of it, why include parent to child in this documentation when the ontology does not include  it?
If we do want them, then make it more obvious that they are being left out

Comment (rjyounes) [reply docs/gistStyleGuide.md:262]: The purpose of the examples is to illustrate how the principle applies to the choices we made.

Comment (rjyounes) [inline docs/gistStyleGuide.md:262]:
```suggestion
| Child to Parent | Parent to Child |
```

File: docs/gistStyleGuide.md (hunk 1 above)
Comment (rjyounes) [reply docs/gistStyleGuide.md:121]: Is this comment out of date? I've changed it to organizations.

**Comment by rjyounes** [comment]
Sorry about this, the release notes here actually go with a different PR which has been merged. The release notes for this PR got merged with that PR. So I'm leaving these release notes to merge in with this PR.

**Pull request merged**